pkg/runner/cluster_k8s.go

```diff
@@ -646,3 +647,28 @@ func (c *ClusterK8sRunner) maxPods() (int, error) {
 
 	return pods, nil
 }
+
+// If the testground daemon has access to enough environment variables, we can terminate using kops
+// if not, then print a messgae about how to do it.
+// I would like to note that when the daemon returns to the client, the cluster may not be deleted
+// yet. This means it will report to the client a success, even if kops is not able to complete the
+// deletion.
+func (c *ClusterK8sRunner) TerminateAll() error {
```

Comment:
I am not sure we want `testground` to have control over infrastructure.

I don't think we want to abstract `kops` or `terraform` or any similar tool inside Testground.

Testground should assume that infrastructure and dependencies are setup correctly. The border between what is Testground and what are dependencies is a bit vague, but I'd say cloud resources are not part of Testground.